zeroconf/__init__.py

```diff
@@ -905,13 +899,7 @@ def add_answer_at_time(self, record: Optional[DNSRecord], now: Union[float, int]
         """Adds an answer if it does not expire by a certain time"""
         if record is not None:
 
-            """Only support for unique answers"""
-            if (
-                record.type == _TYPE_TXT
-                or record.type == _TYPE_SRV
-                or record.type == _TYPE_A
-                or record.type == _TYPE_AAAA
-            ):
+            if self.is_type_unique(record.type):
                 assert record.unique
```

Comment:
We could put the assert in the function/method too and call it `assert_unique_used_correctly` or something I think.

Reply:
I considered it, but thought it was more readable as I have it. I pondered it for a while and decided the readability was better than saving the one line. I'd appreciate your view to be honest and could have gone either way.

Reply:
That's fair, looks good either way.